plugin/tools.py

```diff
@@ -342,7 +372,7 @@ def seconds_from_string(time_str):
         return int(h) * 3600 + int(m) * 60 + int(s)
 
     @staticmethod
-    def get_position_status(point, view, settings):
+    def get_pos_status(point, view, settings):
```

Comment:
I hate them too :smile: . The only thing I hate more is to break lines in every single usage of the function. This forced me to break many lines, which for me hindered readability. So I introduced this. I guess I can live with a couple of shortenings like `pos` or `async` :smile: